README.md

```diff
@@ -1,12 +1,36 @@
 # hie-bios
 
+- [hie-bios](#hie-bios)
+  - [Explicit Configuration](#explicit-configuration)
+    - [Stack](#stack)
+      - [Debugging a `stack` cradle](#debugging-a-stack-cradle)
+      - [Ignoring directories](#ignoring-directories)
+      - [Internal Libraries](#internal-libraries)
+    - [Cabal](#cabal)
+      - [Debugging a `cabal` cradle](#debugging-a-cabal-cradle)
+      - [Ignoring directories](#ignoring-directories-1)
+    - [Bios](#bios)
+    - [Direct](#direct)
+      - [Debugging a `direct` cradle](#debugging-a-direct-cradle)
+    - [None](#none)
+  - [Multi-Cradle](#multi-cradle)
+  - [Cradle Dependencies](#cradle-dependencies)
+  - [Configuration specification](#configuration-specification)
+  - [Testing your configuration](#testing-your-configuration)
+  - [Implicit Configuration](#implicit-configuration)
+    - [Priority](#priority)
+    - [`cabal-install`](#cabal-install)
+    - [`stack`](#stack-1)
+    - [`bios`](#bios-1)
+  - [Supporting Bazel and Obelisk](#supporting-bazel-and-obelisk)
+
 `hie-bios` is the way to specify how
-[`hie`](https://github.com/haskell/haskell-ide-engine) and
-[`ghcide`](https://github.com/digital-asset/ghcide) sets up a GHC API session.
+[`haskell-language-server`](https://github.com/haskell/haskell-language-server)
+and [`ghcide`](https://github.com/digital-asset/ghcide) set up a GHC API session.
 
 Given a Haskell project that is managed by Stack, Cabal, or other package tools,
-`hie` needs to know the full set of flags to pass to GHC in order to build the
-project. `hie-bios` satisfies this need.
+`haskell-language-server` needs to know the full set of flags to pass to GHC in
+order to build the project. `hie-bios` satisfies this need.
```

Comment:
Just to emphasise that the build tools need to do more than just return the flags, but also properly set up the environment by building deps etc
```suggestion
`haskell-language-server` needs to know the full set of flags to pass to GHC in
order to build the project. These flags might contain some compilations options
like `-O2`, but a lot of the time they are package dependencies such as
`-package-id directory-1.3.6.0`, which also need to be built beforehand.
`hie-bios` satisfies both these needs.
```